cmd/init.go

```diff
@@ -105,6 +105,8 @@ func initProvider(homedir string) {
 		MetaAddr:       core.DefaultMetaAddr,
 		PrivateKeyFile: privateKeyPath,
 		PublicKeyFile:  publicKeyPath,
+		SpaceAvail:     107374182400, //100 gB
```

Comment:
Something to think about: when the user first sets up their provider instance, how will we allow them to configure it before registering it with the metaserver?

Reply:
One option (somewhat naive in that it requires a post and then a subsequent put for config)
1. Nav to provider tab
2. Click start providing 
3. Present Config Modal
4. Save Config clicked
5. run "skybin provider"
6. /postInfo (with payload from the config modal)
7. /postinfo will trigger a metaserver update